app/controllers/region_admin/donors_controller.rb

```diff
@@ -0,0 +1,36 @@
+class RegionAdmin::DonorsController < ApplicationController
+  before_filter :authenticate_volunteer!
+  before_filter :authorize_region_admin!
+
+  def index
+    @locations = regional_donors
+    @regions   = available_regions
+  end
+
+  private
+
+  def regional_donors
+    Location.active.donors.regional(region_ids)
```

Comment:
This isn't quite correct - needs to be updated to only select locations in regions where the current volunteer is an admin. 

Updating.